### DISCUSSION TITLE
What is correct way using Interpolation when resizing image?

### DISCUSSION THREAD

**Discussion by tomasr78** [dlemstra/Magick.NET#850] [Help]
I am converting the PDF to Image and would like to use interpolation when resizing the image but setting MagickImage.Interpolate property to any value do not change anything. 

```
 var magickReadSettings = new MagickReadSettings
            {
                ColorSpace = ColorSpace.sRGB
            };

            if (!string.IsNullOrEmpty(password))
            {
                magickReadSettings.Defines = new PdfReadDefines
                {
                    Password = password
                };
            }

            using (var image = new MagickImage())
            {

                image.Read(fileName, magickReadSettings);

                
                    var imageWidth = 1000;
                    var imageHeight = 1000;
                    var size = new MagickGeometry(imageWidth, imageHeight)
                    {
                        IgnoreAspectRatio =true,
                        Greater = true
                    };
                    image.Interpolate = PixelInterpolateMethod.Spline;
                    image.Resize(size);
                }
}
```
I also found `MagickImage.InterpolativeResize` method but not sure is it what I need. Also, `MagickGeometry` class has `IgnoreAspectRatio` and `Greater` properties and  `MagickImage.InterpolativeResize` not.

**Comment by dlemstra** [comment]
What happens when you set `Interpolate` of the `PdfReadDefines` to `true`? And do you have a visual example of what issue you are trying to resolve?

**Reply by tomasr78** [reply]
If I would use PdfReadDefines.Interpolate then where I should set the geometry of the image? The `PdfReadDefines` has `FitPage `property, should I use it to scale the image when combining with `PdfReadDefines.Interpolate`?

Now I use `MagickImage.Resize(size)` method to scale images but I think I could use resizing in GhostScript early with PdfReadDefines.Interpolate.
